front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -260,7 +260,7 @@ export function AgentMessage({
   }, [
     agentMessageToRender.content,
     agentMessageToRender.status,
-    actionToRender,
+    agentMessageToRender.actions.length,
```

Comment:
This might not work as actions "make progress" no?

Reply:
It does work for this hook (which doesn't care about content)